sklearn/linear_model/coordinate_descent.py

```diff
@@ -592,7 +603,18 @@ class ElasticNet(LinearModel, RegressorMixin):
         RandomState instance used by `np.random`. Used when ``selection`` ==
         'random'.
 
+    precompute : True | False | array-like
+        Only if ``solver='cd'``.
+        Whether to use a precomputed Gram matrix to speed up
+        calculations. The Gram matrix can also be passed as argument.
+        For sparse input this option is always ``True`` to preserve sparsity.
+
+    positive : bool, optional
+        Only if ``solver='cd'``.
```

Comment:
maybe "used only if ...", or "relevant only if ..."?

Reply:
How about:

precompute : True | False | array-like
relevant only if ``solver='cd'``
...

positive : bool, optional
If ``solver='cd'`` and set to ``True``, forces the coefficients to be
        positive. ``solver='saga'`` can only be used if set to ``False`` as
        it can't enforce positive coefficients.